qa-heal.planx-pla.net/portal/gitops.json

```
@@ -499,9 +499,9 @@
     ]
   },
   "resourceBrowser": {
-    "title": "Static HEAL Tutorial Notebooks",
+    "title": "Example Analyses of Available Datasets",
     "public": true,
-    "description": "The Jupyter notebooks contained in this notebook viewer pull data from various sources to generate and output useful tables, charts, graphs, and models. Each notebook is static, meaning the data being used by the notebooks is not updated in real time. These notebooks are also available in the Gen3 Tutorials Workspace. Please note these notebooks are meant for illustrative purposes only, and are not meant to portray peer-reviewed research.",
+    "description": "The Jupyter notebooks contained in this notebook viewer pull data from various sources to generate and output useful tables, charts, graphs, and models. Each notebook is static, meaning the data being used by the notebooks is not updated in real time. These notebooks are also available in the Gen3 Tutorials Workspace. Please note these notebooks are meant for illustrative purposes only, and are not meant to portray peer-reviewed research. <a href="https://qa-heal.planx-pla.net/newly-available-datasets">View the latest studies</a> to share their data so that they are accessible on the HEAL Data Platform.",
```

```suggestion
    "description": "The Jupyter notebooks contained in this notebook viewer pull data from various sources to generate and output useful tables, charts, graphs, and models. Each notebook is static, meaning the data being used by the notebooks is not updated in real time. These notebooks are also available in the Gen3 Tutorials Workspace. Please note these notebooks are meant for illustrative purposes only, and are not meant to portray peer-reviewed research. <a href=\"https://qa-heal.planx-pla.net/newly-available-datasets\">View the latest studies</a> to share their data so that they are accessible on the HEAL Data Platform.",
```
need to escape these double quotes